docs/running-a-node/ledger-management.md

```diff
@@ -0,0 +1,188 @@
+1. **Review the differences in representative voting weights by running the [`--compare_rep_weights`](/commands/command-line-interface/#-compare_rep_weights) CLI command** with the new ledger in the default data folder (old ledger backed up) or in a different data folder by using the optional `--data_path` argument. This will compare the new ledger voting weights against the hardcoded values in the node (set at the time of release). See the [CLI command]((/commands/command-line-interface/#-compare_rep_weights)) for details on the output with special attention paid to entries in the `outliers` and `newcomers` sections. By inspecting those addresses in public explorers such as [Nanocrawler.cc](https://nanocrawler.cc), this can help to determine if voting weight may have been manipulated in the downloaded ledger.
+
+If you need support with this process or need help in evaluating some of the CLI command results, join the [Node and Representative Management category](https://forum.nano.org/c/node-and-rep/8) on the [Nano Forums](https://forum.nano.org).
+
+### Confirmation data
+Within each account on the ledger a confirmation height is set. This indicates the height of the last block on that chain where quorum was observed on the network. This is set locally by the node and a new ledger file may include this information with it. If the ledger is from a trusted source this confirmation data can be kept, which will save bandwidth and resources on the network by not querying for votes to verify these confirmations.
+
+If confirmation data for the ledger is not trusted the [--confirmation_height_clear](/commands/command-line-interface/#-confirmation_height_clear) CLI can be used to clear these out.
+
+---
+
+## Updating the node
+
+Occasionally, updating to the [latest node version](/releases/node-releases/#current-release) requires upgrading the existing ledger which can have the following effects:
+
+- Significant downtime, from a few minutes to several hours, during which the node RPC is not accessible and no voting occurs. The upgrade is especially slower if the ledger is not on an SSD.
+- Temporary increased disk space usage - up to 3x the current ledger size in total (e.g. 60GB for a 20GB ledger)
+
+In order to minimize downtime, consider performing the update in a different machine, and replacing the [ledger file](#ledger-file) once complete. Note the following instructions, where **Machine A** has the node and ledger, and **Machine B** will be updating it.
+
+1. Create a directory `/home/<user>/Nano_Update` on Machine B.
+1. Stop the node on Machine A.
+1. Copy the `data.ldb` [file](#ledger-management) from Machine A to `/home/<user>/Nano_Update/data.ldb` on Machine B.
+1. Start the node again on Machine A.
+1. Download the [latest node version](https://github.com/nanocurrency/nano-node/releases/latest). For the purposes of this guide, using a binary is easier.
```

Comment:
```suggestion
1. Download the [latest node version](/releases/node-releases/#current-release). For the purposes of this guide, using a binary is easier.
```

I wasn't sure when I wrote this guide, but do we want to link to the docs instead?